### PR TITLE
105287 Read messages from tie qa

### PR DESCRIPTION
**Pull request by jsoi-eq**
Initial code for reading messages from TIE queue.

[AB#105287](https://statoildeveloper.visualstudio.com/b6d97460-2e53-47ae-aad4-b41396d90828/_workitems/edit/105287)

### CODE REVIEW

File: src/Equinor.ProCoSys.Completion.WebApi/TieImport/Adapter/Tie1MessageHandler.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        CancellationToken stoppingToken)
+        => Task.FromResult(HandleMessage(message));
+
+    public Task Init(TieAdapterConfig config, TieAdapterPartitionConfig partition) => Task.CompletedTask;
+
+    private MessageHandleResult<Tie1Receipt> HandleMessage(Tie1Message message)
+    {
+        //TODO: 105593 Add custom application insights tracking 
+        //_telemetryHelper.TrackMessageReceivedEvent(message.Message);
+        _logger.LogInformation($"Got message with GUID={message.Message.Guid} ({message.Message.Site})");
```
Comment (TordJoranger) [inline src/Equinor.ProCoSys.Completion.WebApi/TieImport/Adapter/Tie1MessageHandler.cs:36]: compile time :)?

**Review by TordJoranger — APPROVED** [review]
How does it work if code here throws an exception? Does the rest of the application carry on as usual? (desired behavior)

File: src/Equinor.ProCoSys.Completion.WebApi/TieImport/Adapter/Tie1MessageHandler.cs (hunk 1 above)
Comment (jsoi-eq) [reply src/Equinor.ProCoSys.Completion.WebApi/TieImport/Adapter/Tie1MessageHandler.cs:36]: ??

**Comment by jsoi-eq** [comment]
> How does it work if code here throws an exception? Does the rest of the application carry on as usual? (desired behavior)
Good question, I have thought about it myself, we don't want Import to take down Completion API.

**Pull request merged**